python/src/uagents/config.py

```diff
@@ -92,7 +92,7 @@ def parse_agentverse_config(
     if "://" in base_url:
         protocol, base_url = base_url.split("://")
     protocol = protocol_override or protocol or "https"
-    http_prefix = "https" if protocol in {"wss", "https"} else "http"
+    http_prefix = "https" if protocol == "https" else "http"
```

Comment:
I can we can leave this the way it was to make it easier to reactivate if we want.